neo/rawio/spikegadgetsrawio.py

```diff
@@ -183,10 +353,10 @@ def _parse_header(self):
                 self.selected_streams = [self.selected_streams]
             assert isinstance(self.selected_streams, list)
 
-            keep = np.isin(signal_streams['id'], self.selected_streams)
+            keep = np.in1d(signal_streams["id"], self.selected_streams)
```

Comment:
```suggestion
            keep = np.isin(signal_streams["id"], self.selected_streams)
```

`in1d` will be deprecated and removed in future NumPy.